_data/master/navbars/networking.yml

```diff
@@ -63,4 +63,4 @@ toc:
   - title: Kuryr
```

Comment:
Don't we need a new entry in this file for the `ip-autodetection.md` file this PR introduces?

Reply:
oops. Fixed.